pegasus/sites.v3/code.org/public/educate/educatorinfo.md

```diff
@@ -1,6 +1,6 @@
 ---
```

Comment:
Is this the only content on this page?
<img width="321" alt="screen shot 2017-04-03 at 1 28 59 pm" src="https://cloud.githubusercontent.com/assets/12300669/24630436/96361c40-1871-11e7-95c8-ad51828065ae.png">